subgraphs/_reference_/src/prices/common/constants.ts

```diff
@@ -92,28 +202,32 @@ UNISWAP_CONTRACT_ADDRESSES.set(
 export const YEARN_LENS_CONTRACT_ADDRESS = new Map<string, string>();
 YEARN_LENS_CONTRACT_ADDRESS.set(
   "mainnet",
-  "0x83d95e0d5f402511db06817aff3f9ea88224b030"  // YEARN_LENS_ORACLE_ADDRESS
+  "0x83d95e0d5f402511db06817aff3f9ea88224b030", // YEARN_LENS_ORACLE_ADDRESS_MAINNET
+);
+YEARN_LENS_CONTRACT_ADDRESS.set(
+  "fantom",
+  "0x0000000000000000000000000000000000000000", // YEARN_LENS_ORACLE_ADDRESS_FANTOM
+);
+YEARN_LENS_CONTRACT_ADDRESS.set(
+  "arbitrum-one",
+  "0x0000000000000000000000000000000000000000", // YEARN_LENS_ORACLE_ADDRESS_ARBITRUM_ONE
 );
 
 ///////////////////////////////////////////////////////////////////////////
 ///////////////////////////// CHAINLINK CONTRACT //////////////////////////
 ///////////////////////////////////////////////////////////////////////////
 
-export const CHAIN_LINK_USD_ADDRESS = Address.fromString(
-  "0x0000000000000000000000000000000000000348"
-);
+export const CHAIN_LINK_USD_ADDRESS = Address.fromString("0x0000000000000000000000000000000000000348");
 
 export const CHAIN_LINK_CONTRACT_ADDRESS = new Map<string, Address>();
-CHAIN_LINK_CONTRACT_ADDRESS.set(
-  "mainnet",
-  Address.fromString("0x47Fb2585D2C56Fe188D0E6ec628a38b74fCeeeDf")
-);
+CHAIN_LINK_CONTRACT_ADDRESS.set("mainnet", Address.fromString("0x47Fb2585D2C56Fe188D0E6ec628a38b74fCeeeDf"));
+CHAIN_LINK_CONTRACT_ADDRESS.set("fantom", Address.fromString("0x0000000000000000000000000000000000000000"));
+CHAIN_LINK_CONTRACT_ADDRESS.set("fantom", Address.fromString("0x0000000000000000000000000000000000000000"));
 
 ///////////////////////////////////////////////////////////////////////////
 ///////////////////////////////// HELPERS /////////////////////////////////
 ///////////////////////////////////////////////////////////////////////////
 
-export const WHITELIST_TOKENS_FORMAT = new TypedMap<string, Address>();
 export const WHITELIST_TOKENS_LIST: string[] = [

```

Comment:
I'm assuming this is needed because `TypedMap` don't have an iterator?

Reply:
yup, correct